modules/standard/Path.chpl

```diff
@@ -62,18 +62,78 @@
 */
 module Path {
```

Comment:
`normPath` should probably be added to [the table of contents](https://chapel-lang.org/docs/master/modules/standard/Path.html#path-computations) above.